Jellyfin.Server/Infrastructure/SymlinkFollowingPhysicalFileResultExecutor.cs

```diff
@@ -101,7 +101,7 @@ protected override Task WriteFileAsync(ActionContext context, PhysicalFileResult
                 count: null);
         }
 
-        private async Task SendFileAsync(string filePath, HttpResponse response, long offset, long? count)
+        private async Task SendFileAsync(string filePath, HttpResponse response, long offset, long? count, CancellationToken cancellationToken = default)
```

Comment:
I think we can just use the `response.HttpContext.RequestAborted` cancellation token instead of allowing one to be passed in?

Reply:
I don't know enough about C# to answer this. Being honest, I've just copied from https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Extensions/src/SendFileResponseExtensions.cs#L85-L110, tested with a local Docker build, and created a PR.

Feel free to make any changes you think are needed.

Reply:
Ah perfect, thanks for providing source